### PR TITLE
Fix section case

### LINKED ISSUE #99: `section:` names do not preserve case
I am importing a few `section`'s like so:
```
plugins:
  - multirepo:
      cleanup: true
      keep_docs_dir: true
      repos:
        - section: Messaging
          import_url: git@gitlab.redacted.com:cdte/docs/lightspeed/lightspeed-docs-messaging.git?branch=main&edit_uri=/edit/main/docs/
        - section: Product APIs
          import_url: git@gitlab.redacted.com:cdte/docs/lightspeed/lightspeed-docs-api.git?branch=main&edit_uri=/edit/main/docs/
```
for `section: Product APIs`, I have also tried it `section: 'Product APIs'`, it is not preserving capitalization as shown here:
Capital

How can I have `section: Product APIs` show up as `Product APIs`, keeping capital/lower case letters as I have it in `mkdocs.yml`?

### PR DESCRIPTION
**Pull request by cloudyparts**
This request disables slugify default lowercase option and honors desired section casing in config.

Allows mkdocs to use folder names to influence navigation style.

Fixes #99

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "mkdocs-multirepo-plugin"
-version = "0.6.3"
+version = "0.6.4"
```
Comment (jdoiro3) [inline pyproject.toml:3]: Can you remove this change? Wanting to combine a few other PRs into one new release. CI automatically generates a release based on this value.

**Review by jdoiro3 — CHANGES_REQUESTED** [review]
Thanks for contributing! Will merge once you make the change to the version number.

File: pyproject.toml (hunk 1 above)
Comment (cloudyparts) [reply pyproject.toml]: removed as requested.  Thanks for the plugin!

**Review by jdoiro3 — APPROVED** [review]
🚀

**Pull request merged**